infrastructure/kube/keep-dev/tbtc-dapp-deployment.yaml

```diff
@@ -25,3 +25,16 @@ spec:
         image: gcr.io/keep-dev-fe24/tbtc-dapp
         ports:
           - containerPort: 80
+        env:
+        # TODO: ETH_WS_URL needs to be using wss:// instead of ws://, otherwise
+        # insecure connection will be blocked.
+        - name: ETH_WS_URL
+          valueFrom:
+            configMapKeyRef:
+              name: eth-network-internal
+              key: ws-url
+        - name: ETH_CHAIN_ID
```

Comment:
I think we need to update this name to match when we use this on this [line](https://github.com/keep-network/tbtc-dapp/blob/1d8456164862ed6af494828caf369d425130204d/src/components/lib/ConnectWalletDialog.js#L7), or that line needs to be updated to match this